modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1233,6 +1233,39 @@ Use `DATABASE *` without the parameter to revoke the privilege on all databases.
 
 Topology notifications provide additional information related to managing databases and servers.
 
+[#_neo_clientnotification_cluster_cordonedserversexistedduringallocation]
+=== CordonedServersExistedDuringAllocation
+.Notification details
+[cols="<1s,<4"]
+|===
+|Code
+m|Neo.ClientNotification.Cluster.CordonedServersExistedDuringAllocation
+|Title
+a| Cordoned servers existed when making an allocation decision.
+|Severity
+m|INFORMATION
+|Category
+m|TOPOLOGY
+|===
+
+.Cordoned servers existed during an allocation decision
+====
+This notification will be returned by any command that triggers an allocation decision. Examples are `CREATE DATABASE`, `ALTER DATABASE`, `DEALLOCATE DATABASES FROM SERVER`, `ALTER DATABASE`.
+Note, `REALLOCATE DATABASES` requires there to be no cordoned servers to be executed successfully thus will never return this notification.
+
+For example, consider a cluster with three servers, of which server `123e4567-e89b-12d3-a456-426614174000` has been cordoned using the `dbms.cluster.cordonServer` procedure. Then the below command will return this notification.
+
+Command::
++
+[source, cypher]
+----
+CREATE DATABASE foo TOPOLOGY 2 PRIMARIES
+----
+
+Description of the returned code::
+The following server(s) are cordoned `123e4567-e89b-12d3-a456-426614174000`. This can impact allocation decisions.
```

Comment:
```suggestion
Server(s) `123e4567-e89b-12d3-a456-426614174000`  is cordoned. 
This can impact allocation decisions.
```

Reply:
A singular vs plural battle 😆  - Server(s) _is_ or _are_ cordoned?

Reply:
Tbh the message could change depending on whether there is 1 server or 2+ servers? Or is this not advisable?

Reply:
I am not sure if it can change. It just looked a bit odd to me using plural with one server. But I am also fine with:
"Server(s) `123e4567-e89b-12d3-a456-426614174000`  are cordoned. "